app/src/main/res/layout/activity_about.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I see you are using a constraint layout here, but it's capabilities are not used anywhere in this file. Consider replacing it with a FrameLayout instead.